src/content/architecture/globalnet/_index.en.md

```diff
@@ -3,32 +3,31 @@ title = "Globalnet Controller"
 date = 2020-02-19T21:08:37+01:00
 weight = 4
 +++
+
 ## Introduction
 
-Submariner is a tool built to connect overlay networks of different
-Kubernetes clusters. These clusters can be on different public clouds
-or on-premise. An important use case for Submariner is to connect disparate
-independent clusters into a single cohesive multi-cluster fleet.
+Submariner is a tool built to connect overlay networks of different Kubernetes clusters. These clusters can be on different public clouds or
+on-premise. An important use case for Submariner is to connect disparate independent clusters into a single cohesive multi-cluster cluster
+set.
```

Comment:
```suggestion
on-premise. An important use case for Submariner is to connect disparate independent clusters into a single cohesive set.
```

Reply:
Maybe "single cluster set"? I was trying to use SIG MC's new term.

Reply:
Made an attempt in a new commit.